zeppelin-web-angular/src/app/core/paragraph-base/paragraph-base.ts

```diff
@@ -54,6 +55,7 @@ export abstract class ParagraphBase extends MessageListenersManager {
 
   // Initialized by `ViewChildren` in the class which extends ParagraphBase
   notebookParagraphResultComponents!: QueryList<NotebookParagraphResultComponent>;
+  notebookParagraphCodeEditorComponent!: NotebookParagraphCodeEditorComponent;
```

Comment:
I think this non-null assertion should be removed and changed to optional. I found it to be `undefined` when `paragraph.config.editorHide === true` in the `%md` editor case. Maybe you could check some errors after running `%md` interpreter
I mistakenly added non-null assertion for `notebookParagraphCodeEditorComponent` in `NotebookParagraphComponent` in a previous commit, but that was incorrect. Sorry for the confusion.